.github/workflows/publish-immutable-actions.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Publish Immutable Action Version'
+
+on:
+  release:
+    types: [created]
```

Comment:
Should we use `release` `published` instead of `created`?

https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=published#release

Users may create a draft release and not want others to depend on it yet.